ml_metadata/proto/__init__.py

```diff
@@ -12,33 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """ML Metadata proto module."""
-from ml_metadata.proto import metadata_store_pb2
-from ml_metadata.proto import metadata_store_service_pb2
-from ml_metadata.proto import metadata_store_service_pb2_grpc
 
 # Connection configurations for different deployment.
-ConnectionConfig = metadata_store_pb2.ConnectionConfig
-MetadataStoreClientConfig = metadata_store_pb2.MetadataStoreClientConfig
+from ml_metadata.proto.metadata_store_pb2 import (
+    ConnectionConfig,
+    MetadataStoreClientConfig,
+)
 
 # ML Metadata core data model concepts.
-Artifact = metadata_store_pb2.Artifact
-Execution = metadata_store_pb2.Execution
-Context = metadata_store_pb2.Context
-
-Event = metadata_store_pb2.Event
-Attribution = metadata_store_pb2.Attribution
-Association = metadata_store_pb2.Association
-ParentContext = metadata_store_pb2.ParentContext
-
-ArtifactType = metadata_store_pb2.ArtifactType
-ExecutionType = metadata_store_pb2.ExecutionType
-ContextType = metadata_store_pb2.ContextType
-
-FakeDatabaseConfig = metadata_store_pb2.FakeDatabaseConfig
-MySQLDatabaseConfig = metadata_store_pb2.MySQLDatabaseConfig
-SqliteMetadataSourceConfig = metadata_store_pb2.SqliteMetadataSourceConfig
+from ml_metadata.proto.metadata_store_pb2 import (
+    Artifact,
+    Execution,
+    Context,
+    Event,
+    Attribution,
+    Association,
+    ParentContext,
+    ArtifactType,
+    ExecutionType,
+    ContextType,
+    FakeDatabaseConfig,
+    MySQLDatabaseConfig,
+    SqliteMetadataSourceConfig,
+)
+from ml_metadata.proto import metadata_store_service_pb2
+from ml_metadata.proto import metadata_store_service_pb2_grpc
 
-del metadata_store_pb2
 del metadata_store_service_pb2
 del metadata_store_service_pb2_grpc
```

Comment:
If you're going to remove the `del metadata_store_pb2` line, can you also remove these other two?

I'm assuming this is to "un-import" the imported protobuf modules...but I'm not clear on why you would want this. If the goal is to re-export certain symbols, `from <module> import <symbol>` is sufficient. For posterity, `del <module>` doesn't "un-import" the module - it persists elsewhere. See https://mail.python.org/pipermail/tutor/2006-August/048596.html for details.

Reply:
I removed `del metadata_store_pb2` because as it is currently written, it is not imported directly anymore. It just has member objects imported. I left the other two `del` statements because they are not used to import member objects. My preference is to leave them as they are and revisit them in the future.